libcontainer/specconv/example.go

```diff
@@ -162,6 +162,10 @@ func ToRootless(spec *specs.Spec) {
 	var namespaces []specs.LinuxNamespace
 
 	// Remove networkns from the spec.
+	//
+	// TODO: removal of networkns should be optional,
```

Comment:
FYI.  When someone puts TODOs in PRs, it makes reviewers nervous.  

todo when?
by who? 
what is the issue now? 
what breaks while this isn't done?